app/models/internal_attachment.rb

```diff
@@ -13,11 +13,11 @@ def initialize(args = {})
     args.each { |k, v| instance_variable_set("@#{k}", v) }
   end
 
-  def method_missing(method_name)
+  def method_missing(method_name, *arguments, &block)
```

Comment:
Going to leave this for now. It's a helper we can refactor away in a later PR, but this diff is a small bugfix I found in how it was setup and used earlier.